apps/service_define/lib/define/typespec_analysis.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Define.TypespecAnalysis do
+  import Logger
+
+  def find() do
```

Comment:
```suggestion
  def find do
```
Also add a spec, please.